File: server/app/models/enums/ground_vehicle_types.py
from enum import Enum


class GroundVehicleTypes(str, Enum):
    TUG = "Tug"
    REFUELER = "Refueler"
    LOADER = "Loader"
    CATERING = "Catering"
    DE_ICER = "De-Icer"
    PUSHBACK = "Pushback"
    BUS = "Bus"

File: server/src/main/java/com/flightcoordinator/server/enums/GroundVehicleTypes.java
package com.flightcoordinator.server.enums;

// TODO turn into an endpoint

public enum GroundVehicleTypes {
  TUG("Tug"),
  REFUELER("Refueler"),
  LOADER("Loader"),
  CATERING("Catering"),
  DE_ICER("De-Icer"),
  PUSHBACK("Pushback"),
  BUS("Bus");

  public final String type;

  private GroundVehicleTypes(String type) {
    this.type = type;
  }
}
